src/main/java/ui/issuepanel/PanelControl.java

```diff
@@ -336,4 +339,31 @@ private void setHvalue(double val) {
             }
         }).collect(Collectors.toList());
     }
+
+    /**
+     *
+     * @return true if there is difference between panels from storage and current panels
+     */
+    public boolean hasUnsavedPanels(){
+        if(!prefs.getLastOpenBoard().isPresent()){
+            return true;
+        }
+        List<PanelInfo> savedPanelInfos = prefs.getBoardPanels(prefs.getLastOpenBoard().get());
+        List<PanelInfo> currentPanelInfos = getCurrentPanelInfos();
+        if (savedPanelInfos.size() != currentPanelInfos.size()) {
+            return true;
+        }
+
+        //savedPanelInfo and currentPanelInfo has to be an exact match (positionally) for them
+        //to be considered equal
+        for (int i = 0; i < savedPanelInfos.size(); i++) {
+            PanelInfo savedPanel = savedPanelInfos.get(i);
+            PanelInfo currentPanel = currentPanelInfos.get(i);
+            if (!savedPanel.getPanelName().equals(currentPanel.getPanelName())
+                    || !savedPanel.getPanelFilter().equals(currentPanel.getPanelFilter())) {
+                return true;
+            }
+        }
+        return false;
+    }
```

Comment:
Can we distill this method into something like `return openBoard.toString().equals(/*string from settings*/)`?
@dariusf do we have a `Board` class that we can use here?